samples/artifacts/EventPublishing/PublishHttpInXmlFormat.siddhi

```diff
@@ -25,29 +24,40 @@ Testing the Sample:
     	b) In the Single Simulation tab of the panel, specify the values as follows:
                 * Siddhi App Name  : PublishHttpInXmlFormat
                 * Stream Name      : SweetProductionStream
-        c) In the name and amount fields, enter 'toffees' and '75.6' respectively and then click Send to send the event. 
+        c) In the name and amount fields, enter 'toffees' and '75.6' respectively and then click Send to send the event.
         d) Send more events as desired.
 
 Viewing the Results:
-    See the output on the terminal.
+    See the output on the terminals.
+        Siddhi Console:
+            INFO {io.siddhi.core.stream.output.sink.LogSink} - PublishHttpInXmlFormat : OutputStream : Event{timestamp=1569929552138, data=[toffees, 75.6], isExpired=false}
 
+        Http-server Terminal:
+            [io.siddhi.distribution.sample.http.server.HttpServerListener] : Received Events: <events><event><name>toffees</name><amount>75.6</amount></event></events>
 Notes:
-    If the message "OutputStream' stream could not connect to 'localhost:8080", it could be due to port 8080
+    If the message stream could not connect to 'localhost:8080", it could be due to port 8080
     defined in the Siddhi application is already being used by a different program. To resolve this issue, do the following:
         1) Stop this Siddhi application (Click 'Run' on menu bar -> 'Stop').
         2) Change the port from 8080 to an unused port in this Siddhi application's source configuration and in the http-server file.
-        3) Start the application and check whether the expected output appears on the console. 
-            
+        3) Start the application and check whether the expected output appears on the console.
+
 */
 
 define stream SweetProductionStream (name string, amount double);
 
 @sink(type = 'http',
 	publisher.url = 'http://localhost:8080/abc',
 	@map(type = 'xml'))
+define stream IntermediateStream (name string, amount double);
+
+@sink(type='log')
 define stream OutputStream (name string, amount double);
 
-@info(name='EventPassThrough')
+@info(name='EventPublishThrough')
 from SweetProductionStream
 select *
+insert into IntermediateStream;
```

Comment:
Whats the use of Intermediate Stream?  Add the sink to the OutputStream.

Reply:
The requested changes are addressed by the commit 6d13673.